From 29e1ce87fd79666e6a8ed668da37469a754ba5c4 Mon Sep 17 00:00:00 2001
From: Jianqun Xu <jay.xu@rock-chips.com>
Date: Mon, 22 Aug 2016 13:03:32 +0800
Subject: [PATCH] ARM64: dts: rk3399-evb3-edp: disable gt9xx

Since edp screen couple with another type of touchscreen,
so disable gt9xx to reduce i2c errors.

Change-Id: I829ae12039ecaea44c4e0734c18a5ebcd41845f8
Signed-off-by: Jianqun Xu <jay.xu@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
index c6056cc23c80..a8e053ace757 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
@@ -115,6 +115,10 @@
 	status = "okay";
 };
 
+&gt9xx {
+	status = "disabled";
+};
+
 &i2c4 {
 	gsl3673: gsl3673@40 {
 		compatible = "GSL,GSL3673";
-- 
2.35.3

